Chart/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Currency Chart</title>
+    <link rel="stylesheet" href="/assets/css/bootstrap.min.css">
+    <link rel="stylesheet" href="/assets/css/style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/lipis/flag-icons@7.2.3/css/flag-icons.min.css"/>
+
+</head>
+<body>
+  <div class="ms-5 mt-5">
+  <p class="fs-1 fw-bold m-0">Forex Exchange</p>
+  <p class="text-muted fw-bold">Check out the current price for a currency pair</p></div>
+    <div id="size" class="container d-flex justify-content-center align-items-center">
+    <div id="dropdown" class="d-flex flex-column gap-4 me-4 col-3">
+    <div class="dropdown">
+        <button class="btn border d-flex dropdown-toggle w-100 height-3 justify-content-between align-items-center" type="button" id="button1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        Select currency
+        </button>
+        <div class=" dropdown-menu flow " id="dropdownItems1" aria-labelledby="dropdownMenuButton">
+
+        </div>
+
+      </div>
+      <div class="dropdown">
+        <button class="btn border d-flex dropdown-toggle w-100 height-3 justify-content-between align-items-center" type="button" id="button2"  data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        Select currency
+        </button>
+        <div class="dropdown-menu flow"  id="dropdownItems2" aria-labelledby="dropdownMenuButton">
+
+        </div>
+    
+      </div></div>
+      <div id="chart" class="card border-0 shadow col-9 mt-5">
+        <div class="d-flex gap-3 align-items-center m-3">
+          <div id="flag1"></div>
+          <div id="flag2"></div>
+          <div class="badge fs-4 bg-light p-1 rounded-pill text-secondary">Forex.com</div>
+          </div>
+        <div class="d-flex align justify-content-between align-items-center">
+        <div class="d-flex fw-bold fs-1 ms-3">
```

Comment:
use default data to get its response and use it because the first rendering not getting data fetched